projects/coincollect/index.js

```diff
@@ -10,8 +10,7 @@ module.exports = {
     incentivized: true,
     polygon: {
         tvl: getUniTVL({ factory: '0x2bc17223A99B6e2857796a5F64A1ED91067b5657', useDefaultCoreAssets: true, fetchBalances: true }),
-        staking: sumTokensExport({owner: MasterChefContract, tokens: [COLLECT], lps: ['0x0cCc84b6506003487AEC687085e82C2f912E607B'], useDefaultCoreAssets: true,  }),
+        staking: stakingUnknownPricedLP(MasterChefContract, COLLECT, "polygon", "0x0cCc84b6506003487AEC687085e82C2f912E607B"),
```

Comment:
`stakingUnknownPricedLP`  is a legacy method and doesnt take into account amount of liquidity,  for example, the pool can have only 1$ worth of tokens and can still be used to as price oracle, thats why we dont use it for new adapters.

sumTokensExport method is bit more careful, think the it allows token value not to exceed 5x (or 10x?) than the liquidity in LPs provided.

Reply:
Solution here might be getting the token listed on coingecko, we pull token prices from there.